.github/workflows/stable-sync.yml

```diff
@@ -119,9 +119,40 @@ jobs:
           # Check if branch exists remotely
           echo "Cleaning up github-tools"
           rm -rf github-tools
+          # Ensure github-tools is not staged or tracked
```

Comment:
I wonder where the `github-tools` file comes from. Maybe instead of introducing all this logic to avoid including it in the commit, we could just add it to `.gitignore`?